var/spack/repos/builtin/packages/pindel/package.py

```diff
@@ -51,9 +50,8 @@ class Pindel(MakefilePackage):
     #
 
     def edit(self, spec, prefix):
-        makefile2 = join_path(self.build_directory, 'Makefile2')
-        copyfile(join_path(self.build_directory, 'Makefile'), makefile2)
-        myedit = FileFilter(makefile2)
+        install('Makefile', 'Makefile2')
```

Comment:
not sure if we really want to use install to copy a file within the build directory

Reply:
I'm currently working on adding `copy` and `copy_tree` functions that will be almost functionally identical to `install` and `install_tree` but semantically will capture what we're going for here.